app/main.py

```diff
@@ -1,3 +1,66 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: [int, "Distance"]) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: [int, "Distance"]) -> "Distance":
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+        else:
+            self.km = self.km + other
+        return self
+
+    def __mul__(self, num: int) -> "Distance":
+        if isinstance(num, Distance):
+            raise TypeError
+        else:
+            self.km = self.km * num
+        return self
+
+    def __truediv__(self, num: int) -> "Distance":
+        if isinstance(num, Distance):
+            raise TypeError
+        else:
+            self.km = round(self.km / num, 2)
+        return self
+
+    def __lt__(self, other: [int, "Distance"]) -> "Distance":
```

Comment:
The return type hint for the comparison methods (`__lt__`, `__gt__`, `__eq__`, `__le__`, `__ge__`) should be `bool` instead of `"Distance"` since these methods return a boolean value.